PWGCF/FemtoDream/femtoDreamProducerTaskV0Only.cxx

```diff
@@ -228,31 +228,36 @@ struct femtoDreamProducerTaskV0Only {
       mRunNumber = bc.runNumber();
     }
 
+    const auto vtxZ = col.posZ();
+    const auto spher = colCuts.computeSphericity(col, tracks);
+    const auto multNtr = col.multNTracksPV();
+    const auto multNtrwithTPC = col.multTPC();
+    const auto multNtracklets = col.multTracklets();
```

Comment:
Same comment as before
Do you plan to simultaneously compare results from different estimators on the same analysis?